### PR TITLE
Add google sign-in docs

### PR DESCRIPTION
**Pull request by Nico-Sanchez**
## Motivation

We didn't have google sign-in docs. We were configuring the Unity Editor asking for someone who knows for help.

## Summary of changes

[Add google sign-in docs](https://github.com/lambdaclass/champions_of_mirra/pull/1853/commits/49a2a169411c22cd340120636c591cb58b9a66df)

## How has this been tested?

This is just documentation. I wrote this based on my experience and several notes I had in notion.

## Checklist
- [ ] I have tested the changes locally.
- [ ] I have tested the whole game after applying the changes, not only the affected areas.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated.

### CODE REVIEW

File: docs/src/google_sign_in.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        - It’s the same as Android Project Name.
+
+When you’re done with the credentials, you must set in the code the following:
+
+- Copy Client ID (Web) and replace it in `client/Assets/Scripts/GoogleSignIn/GoogleSignInController.cs`, `webClientIdGoogle` variable.
+- Under iOS OAuth credentials, download the .plist file and replace its content in `client/Assets/StreamingAssets/GoogleService-Info.plist` .
+
+## Unity Config (project settings)
+
+### Other Settings tab
```
Comment (AngieDutra) [inline docs/src/google_sign_in.md:54]: I would specify this is a tab in 'Player' section

File: docs/src/google_sign_in.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- Scripting backend: IL2CPP.
```
Comment (AngieDutra) [inline docs/src/google_sign_in.md:56]: I would clarify this is in the Android tab since in IOS tab this option is disabled

**Review by AngieDutra — CHANGES_REQUESTED** [review]
Awesome. Just a few comments.
Disclaimer: I didn't test the whole doc, just the player settings part

File: docs/src/google_sign_in.md (hunk 1 above)
Comment (Nico-Sanchez) [reply docs/src/google_sign_in.md:54]: Done!

File: docs/src/google_sign_in.md (hunk 2 above)
Comment (Nico-Sanchez) [reply docs/src/google_sign_in.md:56]: Nice catch! Done!

**Review by AngieDutra — APPROVED** [review]
Awesome and also reviewed the IOS process

**Review by alfopisano — APPROVED** [review]

**Pull request merged**